hadoop-common-project/hadoop-common/src/test/java/org/apache/hadoop/io/erasurecode/TestCodecRawCoderMapping.java

```diff
@@ -150,4 +155,44 @@ public void testIgnoreInvalidCodec() {
             conf, ErasureCodeConstants.XOR_CODEC_NAME, coderOptions);
     Assert.assertTrue(decoder instanceof XORRawDecoder);
   }
+
+  @Test
+  public void testCodecNativeEnabled() {
+    if (!ErasureCodeNative.isNativeCodeLoaded()) {
+      LOG.warn("ISA-L support is not available in your platform.");
+      return;
+    }
+    ErasureCoderOptions coderOptions = new ErasureCoderOptions(
+        numDataUnit, numParityUnit);
+
+    conf.setBoolean(CodecUtil.IO_ERASURECODE_CODEC_NATIVE_ENABLED_KEY,
```

Comment:
Is this necessary? It's the default, so it seems it wouldn't be necessary to set explicitly.